frontend/src/reducers/ActionReducers.ts

```diff
@@ -30,7 +30,7 @@ export default function setAction(state : State, action: Action) {
                 }
             }
         case ActionTypes.addItem:
-            let newItems = state.store.items;
+            let newItems = [...state.store.items];
```

Comment:
```suggestion
            let newItems = [...state.store.items, (action as addItemAction).addedItem];
```
You could also combinr this line and the line below it like so